### DISCUSSION TITLE
vc-imagery-provider-arcgis加载图层，参数中只有最大层级，没有最小层级且无效

### DISCUSSION THREAD

**Discussion by zyg-admin** [zouyaoji/vue-cesium#641] [Q&A]
vc-imagery-provider-arcgis加载图层，参数中只有最大层级，没有最小层级且无效。默认去加载了全部，我图层中是没有19层的，导致404，该如何解决
<img width="311" alt="750ecee40e4ac0b72dd31de39b9d47a" src="https://github.com/user-attachments/assets/28fc9c2e-a649-4e4f-9ef6-f05501dca5c1" />
图片中部分层级的startFile是空的

Cesium.js:76 
 GET http://xxx:8000/arcgis/rest/services/MapServer/tile/19/0/0 404 (Not Found)

**Comment by zouyaoji** [comment]
这应该是cesiumjs本身的限制。
